src/verilog.y

```diff
@@ -343,7 +343,7 @@ static void ERRSVKWD(FileLine* fileline, const string& tokname) {
     fileline->v3error(
         std::string{"Unexpected '"} + tokname + "': '" + tokname
         + "' is a SystemVerilog keyword misused as an identifier."
-        + (!toldonce++ ? "\n" + V3Error::warnMore()
+        + (!toldonce++ ? "\n" + V3Error::s().warnMore()
```

Comment:
I've explained it in more details in my answer above.

There are still some places in the code where we are calling ``V3Error::warnMore()`` outside of ``v3warn`` macros, so I need to have function that doesn't require already locked V3Error mutex.

If you are OK with this, I can create new function that would be current ``V3Error::warnMore()`` (so it would excludes V3Error mutex) and it could be used outside ``v3warn`` macros, then I could create ``V3Error::warnMore()`` function that requires V3Error mutex that would simply call ``s().warnMore()``.

Reply:
I've added ``warnMoreGuarded`` function that excludes V3Error mutex with annotation that this function isn't MT_SAFE, so when it will be used in MT_SAFE context, ``clang_check_annotations`` script will warn about this,

Now ``V3Error::warnMore`` simply calls ``s().warnMore()`` (and requires V3Error mutex).

Reply:
See my note above about magic postprocessing, I think that is a cleaner solution (to the V3Errror users anyways).